pkgs/by-name/db/dbdev/package.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  darwin,
+  fetchFromGitHub,
+  nix-update-script,
+  openssl,
+  pkg-config,
+  rustPlatform,
+  stdenv,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "dbdev";
+  version = "2024-12-13";
```

```suggestion
  version = "0.1.6-unstable-2024-12-13";
```
Standard scheme for pre-release packaging

---

and there's been several commits (but no new tag)